services/movie.py

```diff
@@ -0,0 +1,46 @@
+from typing import List, Optional
+
+from django.db.models import QuerySet
+
+from db.models import Genre, Actor, Movie
+
+
+def get_movies(
+        genres_ids: Optional[List[int]] = None,
+        actors_ids: Optional[List[int]] = None
+) -> QuerySet:
+    queryset = Movie.objects.all()
+
+    if genres_ids:
+        queryset = queryset.filter(genres__id__in=genres_ids).distinct()
+
+    if actors_ids:
+        queryset = queryset.filter(actors__id__in=actors_ids).distinct()
+
+    return queryset
+
+
+def get_movie_by_id(movie_id: int) -> Optional[Movie]:
+    return Movie.objects.get(id=movie_id)
```

Comment:
The `get_movie_by_id` function uses `Movie.objects.get(id=movie_id)`, which will raise a `DoesNotExist` exception if no movie with the given ID is found. Consider using `Movie.objects.filter(id=movie_id).first()` to return `None` instead of raising an exception if the movie does not exist.